frappe/core/api/user_invitation.py

```diff
@@ -0,0 +1,149 @@
+import frappe
+from frappe import _
+from frappe.model.document import Document
+
+
+def _validate_app_name(app_name: str) -> None:
```

Comment:
https://github.com/frappe/erpnext/wiki/Coding-Standards 

Follow this rough ordering guideline for readability.